wgengine/magicsock/magicsock.go

```diff
@@ -1727,7 +1727,16 @@ func (c *Conn) runDerpReader(ctx context.Context, derpFakeAddr netip.AddrPort, d
 		case derp.HealthMessage:
 			health.SetDERPRegionHealth(regionID, m.Problem)
 		case derp.PeerGoneMessage:
-			c.removeDerpPeerRoute(key.NodePublic(m), regionID, dc)
+			switch m.Reason {
+			case derp.PeerGoneReasonDisconnected:
+			case derp.PeerGoneReasonNotHere:
```

Comment:
you know this isn't like C, right? (There's no implicit `fallthrough`)

It's not clear this is intentional. Maybe add a comment like `// Do nothing.` to the first case.

Reply:
Or if you want to do it on both cases, do "case A, B:"

Reply:
Thanks! Comment added.